### DISCUSSION TITLE
after installation: directory listing instead of kimai

### DISCUSSION THREAD

**Discussion by robertmeggle** [kimai/kimai#3184] [On-Premise: Installation & Updates] [answered]
After installation of kimai according to https://www.kimai.org/documentation/installation.html#recommended-setup
... I just see directory listing. 
crosschecked: 
- git clone -b 1.18.1 --depth 1 https://github.com/kevinpapst/kimai2.git
- cd kimai2/
- composer install --no-dev --optimize-autoloader -n
- configured .env file
- bin/console kimai:install -n
last command followed by: Congratulations! Successfully installed Kimai version 1.18.1
But when I call localhost/kimai/kimai2 i just get a directory listing instead of login sceen. 
cat  
Guess something is missing, but no clue what... 

On apache's access log I can see 200 and 304 (ok and not modified). error.log is empty. 
My system: composer ver. 2.2.7, php 8.1.0, Ubuntu Jammy Jellifish (dev branch). 
installed: apache + mysql + php + phpmyadmin
php & mysql seems working fine, because I can administrate my db via phpmyadmin quite well and also an test output of phpinfo() worked.

**Comment by kevinpapst** [comment, marked as answer]
You missed the tiny but important [piece of the puzzle](https://www.kimai.org/documentation/webserver-configuration.html): 

<img width="799" alt="Bildschirmfoto 2022-03-02 um 16 44 02" src="https://user-images.githubusercontent.com/533162/156395979-dadb3ec3-7364-4d70-a501-e50aee88ca6f.png">

**Reply by robertmeggle** [reply]
Thanks - had a wrong path inside my VirtualHost. "public" seems important :-)
